VideoLocker/res/values/strings.xml

```diff
@@ -250,7 +244,10 @@
     <string name="discussion_post_endorsed">Endorsed</string>
     <!-- Used for putting authorLabel into parenthesis e.g. (Community TA) or (Staff) -->
     <string name="discussion_post_author_label_attribution">({text})</string>
-
+    <!-- Label denoting the number of replies on a post e.g. Total: 47 -->
```

Comment:
Maybe replace 'denoting' with 'indicating', which is an easier and more common word.